Minio.Tests/Minio.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net6.0;</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
IMO lets still do 2.1 for Linux @ebozduman

Reply:
ok